module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1248,10 +1248,11 @@ public function placeHold($holdDetails)
     {
         $default_request = $this->config['Holds']['default_request'] ?? 'Hold';
         try {
-            $requiredBy = $this->dateConverter->convertFromDisplayDate(
-                'Y-m-d',
-                $holdDetails['requiredBy']
-            );
+            $requiredBy = isset($holdDetails['requiredBy']) ?
```

Comment:
I'd suggest taking advantage of the recently added requiredByTS field that gets automatically added to holdDetails. It'll allow you to simplify the whole try-catch to this: 
`$requiredBy = !empty($holdDetails['requiredByTS']) ? date('Y-m-d', $holdDetails['requiredByTS']) : null;`

Reply:
Fixed the example above to use gmdate since the the timestamp is always UTC.